frontend/src/features/admin/access/__snapshots__/ManageAccessRequests.test.tsx.snap

```diff
@@ -275,7 +275,7 @@ exports[`Manage access requests Snapshot matches 1`] = `
             Object {
               "display": "flex",
               "flex": "1 0 auto",
-              "minWidth": "780px",
+              "minWidth": "580px",
```

Comment:
Where did this change come from?

Reply:
From removing the **Agency** column from the list view in Manage Access Requests